pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -514,6 +518,36 @@ downloadPhase:
 	}
 }
 
+// isAppAlreadyInstalled checks whether app is already installed on the pod
+func (ctx *localScopePlaybookContext) isAppAlreadyInstalled(appInstallStatusFilePath string) bool {
```

Comment:
As discussed, ideally this check shouldn't be needed for existing pods. To be checked under what exact scenario this is needed?